lambdas/llama-2-13b-chat-llm/template.yml

```diff
@@ -0,0 +1,114 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: QnABot on AWS LLM Plugin for Llama-2-13b-chat model deployed using SageMaker JumpStart (https://aws.amazon.com/blogs/machine-learning/llama-2-foundation-models-from-meta-are-now-available-in-amazon-sagemaker-jumpstart/)
+
+Parameters:
+  SageMakerEndpointName:
+    Type: String
+    Description: Get the SageMaker Endpoint Name for Llama 2 13b Chat LLM Model from Amazon SageMaker console > Choose Inference in the left panel > Choose Endpoints. Refer to this link on how to deploy the Llama-2-chat model in SageMaker JumpStart - https://aws.amazon.com/blogs/machine-learning/llama-2-foundation-models-from-meta-are-now-available-in-amazon-sagemaker-jumpstart/
+    Default: 'jumpstart-dft-meta-textgeneration-llama-2-13b-f'
+
+  SageMakerEndpointArn:
```

Comment:
I see that this parameter is only used for the IAM policy (line 40). Instead of having user to input the ARN as the Cfn parameter, I would suggest to construct the Arn as part of the template itself, using the `SageMakerEndpointName` parameter i.e.

`!Sub arn:${AWS::Partition}:sagemaker:${AWS::Region}:${AWS::AccountId}:endpoint/${SageMakerEndpointName}`

Reply:
Thanks for your feedback. I have updated template.yml. Changes: 1) Removed SageMakerEndpointARN parameter, 2) Updated resource policy to construct the ARN using the SageMakerEndpointName parameter.